components/atoms/button.html

```diff
@@ -24,13 +24,9 @@
         text-align: center;
         cursor: pointer;
     }
-    a.button {
-        text-decoration: none;
-    }
   </style>
 
   <markup>
-    <a href="#" class="button">Link</a>
-    <button class="button">Button</button>
```

Comment:
Don't delete the link button! Also, keep he text generic for the component, like "Loren Ipsum" or "button"